content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,255 @@
+---
+parent: build-apps.html
+targets:
+  - en
+  - ja # TODO: translate this page
+blurb: Build a graphical desktop wallet for the XRPL using JavaScript.
+---
+# Build a Desktop Wallet in JavaScript
+<!-- STYLE_OVERRIDE: wallet -->
+
+This tutorial demonstrates how to build a desktop wallet for the XRP Ledger using the JavaScript programming language, 
+the Electron Framework and various libraries. This application can be used as a starting point for building a more 
+complex and powerful application, as a reference point for building comparable apps, or as a learning experience to 
+better understand how to integrate XRP Ledger functionality into a larger project.
+
+## Prerequisites
+
+To complete this tutorial, you should meet the following requirements:
+
+- You have node.js 14+ installed on your machine. You can get the latest version [here](https://nodejs.org/en/).
```

Comment:
As a best practice, use informative link text rather than "here". I suggest simplifying the line as follows:

```suggestion
- You have [Node.js](https://nodejs.org/) 14+ installed.
```

Reply:
Done